test/runnable/test15371.d

```diff
@@ -0,0 +1,31 @@
+// EXTRA_SOURCES: imports/test15371.d
+
+/*
+TEST_OUTPUT:
+---
+
+---
```

Comment:
Shouldn't there be any output?

Reply:
That was the weird part, I could get the output when I placed an failing assert in the code, but without it, I got nothing.

Reply:
I think `TEST_OUTPUT` is usually used for asserting error messages for failing tests. Is there any difference if you print to stderr?

Reply:
You need to use `pragma(msg,...)` for TEST_OUTPUT as it only listens to output during compilation.